### PR TITLE
add autoActivate argument

### PR DESCRIPTION
**Pull request by li-yq**
Added an `autoActivate` argument so that auto activation of the base environment can be manually disabled.

### CODE REVIEW

File: src/micromamba/devcontainer-feature.json [hunk 1]
```diff
@@ -18,6 +18,11 @@
             "default": false,
             "description": "Reinstall in case Micromamba already exists"
         },
+        "autoActivate": {
+            "type": "boolean",
+            "default": true,
+            "description": "Auto activation of base environment"
+        },
```
Comment (eitsupi) [inline src/micromamba/devcontainer-feature.json:25]: Could you add a new test for `autActivate: false`? (If it is automatically testable)

**Review by eitsupi — COMMENTED** [review]
Thanks for your contribution!

Could you bump the minor version in the json file?

File: src/micromamba/devcontainer-feature.json (hunk 1 above)
Comment (li-yq) [reply src/micromamba/devcontainer-feature.json:25]: I'd like to (and actually has already tried), but I did not figure out how the tests works.

Can you provide any suggestions on where should I start from?

Comment (eitsupi) [reply src/micromamba/devcontainer-feature.json:25]: Write the installation options in a json file and test the expected commands in a shell script.

https://github.com/mamba-org/devcontainer-features/blob/400d53fe9b66ca793215c866021c245b90dcb51c/test/micromamba/scenarios.json#L13-L20

https://github.com/mamba-org/devcontainer-features/blob/400d53fe9b66ca793215c866021c245b90dcb51c/test/micromamba/install-specific-version.sh#L1-L13

Check the Dev Container CLI documentation for more information.
https://github.com/devcontainers/cli/blob/main/docs/features/test.md

Comment (li-yq) [reply src/micromamba/devcontainer-feature.json:25]: Thanks for your help. The test is now added.

**Review by eitsupi — APPROVED** [review]
Looks good.

**Review by maresb — APPROVED** [review]
Yes, this looks great, thanks a lot for the contribution!

**Comment by eitsupi** [comment]
The new version was released. 🎉
https://github.com/mamba-org/devcontainer-features/pkgs/container/devcontainer-features%2Fmicromamba/106515421?tag=1.2.0

**Pull request merged**